services/thermal_mgr/thermal_mgr.c

```diff
@@ -44,17 +45,56 @@ void initThermalSystemManager(lm75bd_config_t *config) {
 error_code_t thermalMgrSendEvent(thermal_mgr_event_t *event) {
   /* Send an event to the thermal manager queue */
 
+  if (thermalMgrQueueHandle == NULL || event == NULL) {
```

Comment:
The queue handle isn't an arg so it should return invalid state instead of invalid arg